setup.cfg

```diff
@@ -1,3 +1,56 @@
+[metadata]
+name = kube-hunter
+description = Kubernetes security weaknesses hunter for humans
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = Aqua Security
+author_email = support@aquasec.com
+url = https://github.com/aquasecurity/kube-hunter
+keywords =
+    aquasec
+    hunter
+    kubernetes
+    k8s
+    security
+license_file = LICENSE
+classifiers =
+    Development Status :: 4 - Beta
+    Environment :: Console
+    License :: OSI Approved :: Apache Software License
+    Natural Language :: English
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Topic :: Security
+
+[options]
+zip_safe = False
+packages = find:
+install_requires =
```

Comment:
do we need to duplicate the requirements.txt here, or we could share this list. there seems to be different approaches to handle this, one is replacing requirements with just `.`

Reply:
Thanks @itaysk 
I feel it is more of a personal choice of how folks like to configure their setup & requirements files. I see the importance of keeping both the `install_requires` list and `requirements.txt` in the project as also mentioned here in the [docs](https://packaging.python.org/discussions/install-requires-vs-requirements/#requirements-files). 

It's not only about duplicating requirements, but it's also about the best practice of listing only those dependencies on which the project minimally depends on. It's also the list that pip runs when the package will be installed. Whereas requirements.txt can be worked towards building a complete Python env for developers and contributors.

Reply:
@itaysk I agree about having two sources of truth is a bad idea since they will inevitably get out of sync.
As for what @vipulgupta2028 mentioned, it is correct that requirements file should make a reproducible python environment.
However, since we don't run a single production-like kube-hunter environment, we don't really need to have a requirements file with strict version specification so using `-e .` fits.

Reply:
Roger that, folks. Implemented the change. Resolving.